migration_guides/upgrading_to_terragrunt_0.12.x.md

```diff
@@ -0,0 +1,174 @@
+# Upgrading to Terragrunt 0.12.x
```

Comment:
In Gruntwork repos, we follow a convention of putting docs files other than the top-level README in a top-level folder called `_docs`. This clearly separates it from code folders. Can you rename?